src/ui/public/autoload/styles.js

```diff
@@ -0,0 +1,2 @@
+const context = require.context('../styles', false, /[\/\\](?!mixins|variables|_|\.)[^\/\\]+\.less/);
+context.keys().forEach(key => context(key));
```

Comment:
yeah, I just don't like passing the other arguments (index and list). This feels concise enough.